src/sdk/project/asset-json.ts

```diff
@@ -53,7 +55,11 @@ export namespace AssetJson {
       },
       category: WearableCategory.schema,
       rarity: Rarity.schema,
-      bodyShape: AssetWearableGender.schema
+      bodyShape: AssetWearableGender.schema,
+      collectionId: {
+        description: 'The collection where the wearable is placed.',
+        type: 'string'
```

Comment:
would it be useful to add a `format: 'uuid'` to this type?

Reply:
yep